ui/vendor/css/bootstrap.css

```diff
@@ -4087,7 +4527,16 @@ select[multiple].input-group-sm > .input-group-btn > .btn {
     margin-right: 15px;
     margin-left: 15px;
   }
-  .navbar-text.navbar-right:last-child {
+}
+@media (min-width: 768px) {
+  .navbar-left {
+    float: left !important;
+  }
+  .navbar-right {
+    float: right !important;
+    margin-right: -15px;
```

Comment:
So I think this new `margin-right: -15px;` is causing our upper right navbars to shrink, including the quick filter. We may need to add our own treeherder.css override with a `margin-right: 0;` if we want to preserve our current navbar appearance and quick filter dimension.

The default bootstrap appearance showing the shrinkage
![navbarright](https://cloud.githubusercontent.com/assets/3660661/8853025/9fa9e466-3126-11e5-9107-6370603c3e3a.jpg)